SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -536,10 +540,13 @@ - (BOOL)sdl_isTCPReady {
     return YES;
 }
 
-#pragma mark - SDLProtocolListener Implementation
+#pragma mark - SDLProtocolDelegate Implementation
 
 // called on transport's thread, notifying that the transport is established
-- (void)onProtocolOpened {
+- (void)protocolDidOpen:(SDLProtocol *)protocol {
+    // The primary transport opened
```

Comment:
```suggestion
```

This is actually checking for the secondary transport, right? As noted by the log below.